.github/workflows/claude.yml

```
@@ -0,0 +1,37 @@
+name: Claude Code
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_review_comment:
+    types: [created]
+  issues:
+    types: [opened, assigned]
+  pull_request_review:
+    types: [submitted, opened, edited]
```

actionlint (1.7.7)

invalid activity type "opened" for "pull_request_review" Webhook event. available types are "dismissed", "edited", "submitted"

(events)